### ISSUE TITLE
Provide more useful error message instead of "undefined local variable or method `flash'"

### ISSUE THREAD

**Issue by inem** [hanami/hanami#859]
I am trying to create some very basic app with Hanami, and recently got this weird error message:
```
undefined local variable or method `flash' #<Web::Controllers::Sessions::Create:0x007fb84ef98068>
```

It took me a while to figure out what is wrong. I forgot to add `include Hanami::Action::Session` to my action, but nothing suggested that I should do so.

I guess a lot of newbies might fall into this trap as well.

My suggestion is to make error message more useful and self-explanatory

**Comment by AlfonsoUceda** [comment]
@inem thanks for the suggestion, I think we should document the behaviour, what do you think @hanami/core?

**Comment by cllns** [comment]
I think we could go a step further and look for `session` and `flash`, and raise errors when those happen with a helpful message saying something like "Did you forget to add `include Hanami::Action::Session` to your action?"

(This would be a change to [`hanami/controller`](https://github.com/hanami/controller))

**Comment by davydovanton** [comment]
@cllns or "For using session add `include Hanami::Action::Session`. For more information check docs: doc_link" :)

**Comment by AlfonsoUceda** [comment]
@wheresmyjetpack don't worry, just open the PR and we discuss there ;)

**Comment by jodosha** [comment]
Implemented by #915
